espnet/nets/pytorch_backend/transformer/utils.py

```diff
@@ -0,0 +1,24 @@
+import torch
+
+"""Unility funcitons for Transformer."""
+
+
+def add_sos_eos(ys_pad, sos, eos, ignore_id):
```

Comment:
As we prefer to tiny modules,  you should name this file as `add_sos_eos` instead of utils.